moban/mobanfile/__init__.py

```diff
@@ -34,10 +34,9 @@ def handle_moban_file_v1(moban_file_configurations, command_line_options):
 
     targets = moban_file_configurations.get(constants.LABEL_TARGETS, [])
     if constants.LABEL_COPY in moban_file_configurations:
-        legacy_copy_targets = handle_copy(
-            merged_options, moban_file_configurations[constants.LABEL_COPY]
+        raise NotImplementedError(
+            "copy targets has been removed. please refer to level 15 in docs"
```

Comment:
Maybe add the version here? For example `Removed since version ...` and also maybe we can give the alternative.

Reply:
Without this PR, it does give depreciation warning.